types/chainid.go

```diff
@@ -15,4 +15,5 @@ const (
 	ChainIDMoonriver  = 1285
 	ChainIDBoba       = 288
 	ChainIDTon        = 607
+	ChainIDZKEVM      = 1422
```

Comment:
No sui here?

Reply:
I didn't find the chain ID here https://github.com/trustwallet/wallet-core/blob/686cdd1c5efaadea7f6deea0d82bb355cb30797e/registry.json#L433 for Sui